### PR TITLE
Update pokecli.py

### PR DESCRIPTION
**Pull request by sohje**
## Short Description:
Check PgoApi version on bot start
## Fixes/Resolves/Closes (please use correct syntax):
- Prevent issues on outdated environments.

Check pgoapi version on startup

### CODE REVIEW

**Comment by rawgni** [comment]
does this check the requirements.txt for version information or pip?

**Comment by sohje** [comment]
We dont have required version of pgoapi in requirements.txt, only commit hash.
A lot of issues appeared on pgoapi_version < 1.1.8

File: pokecli.py [hunk 1]
```diff
@@ -57,6 +57,20 @@
...
+    import pkg_resources
+    pgoapi_version = pkg_resources.get_distribution("pgoapi").version
+    if pgoapi_version < '1.1.8':
+        print "Run following command to get latest update: `pip install -r requirements.txt --upgrade`"
+        sys.exit(1)
+except pkg_resources.DistributionNotFound:
+    print 'Seems you forgot to install python modules.'
+    print 'Run: `pip install -r requirements.txt`'
+    sys.exit(1)
+except ImportError as e:
```
Comment (solderzzc) [inline pokecli.py]: Is this good to bypass missing import?

Comment (sohje) [inline pokecli.py]: It should bypass if import pkg_resources raises ImportError.

Comment (solderzzc) [inline pokecli.py]: oh, I understand. Thanks 
👍

**Comment by solderzzc** [comment]
👍

**Pull request merged**